### PR TITLE
Update mental_health_in_tech.md

### PR DESCRIPTION
**Pull request by cebarks**

### CODE REVIEW

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
-1. Effects of Tech Industry culture on mental health conditions, ways to prevent burnout, and how we can support each other.
+Effects of Tech Industry culture on mental health conditions, ways to prevent burnout, and how we can support each other.
 
 ## Session Outcomes 
-
 1. An understanding of how issues of mental health are addressed, or not addressed, in the industy.
 2. Ways to find support and how they can support each other.
 
-## Session Outline 
+## Supplies
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:17]: Can do!

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 2]
```diff
@@ -1,30 +1,45 @@
...
-**(10 min)** looking forward, and strategies
-**(5 min)** students write in their journal about some of their own strategies and ways they can help others.
+## Session Outline – Total: 70 Minutes
+1. **(20 min)** Gear-up Introduction
+  - **(5 min)** Facilitator Introductions
+    - Personal Stories/Experiences; Why do we think it's important?
+  - **(5 min)** Group Introductions
+    - Name, pronouns, etc.
+    - Why they chose this session?
+  - **(5 min)** Group Norms
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:29]: Great on setting norms.  Have a few in place you want to use!

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 3]
```diff
@@ -1,30 +1,45 @@
...
+    - Why is it important to talk about mental health?
+2. **(10 min)** Pomodoro Break & 3 min Headspace Meditation
+3. **(15 min)** small groups of 3, reading different articles and discussions as groups.
+  - **(8 min)** Read different articles of a group individually
+    1. *[Stress & Anxiety In The Tech Industry](https://risepsychology.com/blog/2017/10/30/stressintech)*
+    2. *[Are You More Than Okay: The State of Mental Health in Tech in 2016](https://modelviewculture.com/pieces/are-you-more-than-okay-the-state-of-mental-health-in-tech-in-2016)*
+    3. *[How should we talk about mental health?](https://ideas.ted.com/how-should-we-talk-about-mental-health/)*
+  - Questions to think about
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:37]: Great on these discussion questions.  Maybe have them written on the board.

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 4]
```diff
@@ -1,30 +1,45 @@
...
+    - What is your biggest take away from the article?
+    - What surprised you the most in the article?
+    - What kind of changes could you make in your own life?
+    - What are some other ideas/strategies for mental health support?
+  - **(7 min)** Disccusions in small groups
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:42]: How do you want to select groups?  Mix them up?  Count off?

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 5]
```diff
@@ -1,30 +1,45 @@
...
+3. **(10 min)** large group discussion
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:43]: Consider how you can hear from multiple voices in the large group discussion.  Maybe ask a few people to share before large group starts?

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 6]
```diff
@@ -1,30 +1,45 @@
...
+4. **(10 min)** looking forward, and strategies
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:44]: What are the strategies you want them to take away? I'd have a suggested set of things in place so you can share those in addition to anything that came up during the session.

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 7]
```diff
@@ -1,30 +1,45 @@
...
+5. **(5 min)** students write in their journal about some of their own strategies and ways they can help others.
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:45]: Great closing!  Maybe have 1-2 people share at the end if you have time.

File: m4_sessions/1808-inning/mental_health_in_tech.md [hunk 8]
```diff
@@ -1,30 +1,45 @@
...
+## Session Outline – Total: 70 Minutes
+1. **(20 min)** Gear-up Introduction
+  - **(5 min)** Facilitator Introductions
+    - Personal Stories/Experiences; Why do we think it's important?
+  - **(5 min)** Group Introductions
+    - Name, pronouns, etc.
+    - Why they chose this session?
+  - **(5 min)** Group Norms
+    - Why is it important to talk about mental health?
+2. **(10 min)** Pomodoro Break & 3 min Headspace Meditation
```
Comment (emhickmann) [inline m4_sessions/1808-inning/mental_health_in_tech.md:31]: I like that you have this!  Maybe you could do a stretch break at the end of the reading or before you close out?

**Review by emhickmann — COMMENTED** [review]
@cebarks Nice work on this session!  Love the articles and the outline.  I have a few specifics for you to consider, but you are in a great place!

File: m4_sessions/1808-inning/mental_health_in_tech.md (hunk 2 above)
Comment (ericweissman) [reply m4_sessions/1808-inning/mental_health_in_tech.md:29]: Below are some norms we will use as a jumping off point and allow others to contribute to!

- Every mistake is a learning experience
- Demonstrate empathy
- Give people space to share experiences
- Don't assume intentions
- Work to understand the perspective of others
- Accept others' lived experiences as truth
- Be at peace with being uncomfortable
- Permission to make mistakes
- Self-care is extremely important and should be on the forefront

File: m4_sessions/1808-inning/mental_health_in_tech.md (hunk 3 above)
Comment (ericweissman) [reply m4_sessions/1808-inning/mental_health_in_tech.md:37]: We will have these discussion questions on the board

File: m4_sessions/1808-inning/mental_health_in_tech.md (hunk 4 above)
Comment (ericweissman) [reply m4_sessions/1808-inning/mental_health_in_tech.md:42]: Count off by 3s

File: m4_sessions/1808-inning/mental_health_in_tech.md (hunk 5 above)
Comment (ericweissman) [reply m4_sessions/1808-inning/mental_health_in_tech.md:43]: We will have chart paper on each table. Have one person writing down some responses from the small group sharing and then ask volunteers to read off a few things from their chart paper.

File: m4_sessions/1808-inning/mental_health_in_tech.md (hunk 6 above)
Comment (ericweissman) [reply m4_sessions/1808-inning/mental_health_in_tech.md:44]: Strategies for Mental Health
- Use Headspace and/or similar apps to help with self-reflection, being mindful, etc.
- Don't be afraid to seek out a mental health professional and be supportive of others who want to meet with a mental health professional
- Build techniques into your work routine to give yourself deliberate breaks/opportunities to disconnect from the work (Pomodoros, change of scenery/get fresh air, etc.)
- Take care of your body in a way that works for you - daily walk, moving around, etc.

File: m4_sessions/1808-inning/mental_health_in_tech.md (hunk 7 above)
Comment (ericweissman) [reply m4_sessions/1808-inning/mental_health_in_tech.md:45]: Yeah, we definitely will!

File: m4_sessions/1808-inning/mental_health_in_tech.md (hunk 8 above)
Comment (ericweissman) [reply m4_sessions/1808-inning/mental_health_in_tech.md:31]: That's a great idea. We will have a short reading/quote prepared or potentially show the power of going outside for 5 minutes for fresh air, weather permitting.

**Pull request merged**